backend/ddang/src/main/java/com/ddang/ddang/chat/application/ChatRoomService.java

```diff
@@ -69,51 +68,33 @@ private void checkAuctionStatus(final Auction findAuction) {
         if (!findAuction.isClosed(LocalDateTime.now())) {
             throw new InvalidAuctionToChatException("경매가 아직 종료되지 않았습니다.");
         }
-        if (findAuction.isDeleted()) {
-            throw new InvalidAuctionToChatException("삭제된 경매입니다.");
-        }
     }
 
     private void checkUserCanParticipate(final User findUser, final Auction findAuction) {
-        if (!isSellerOrWinner(findUser, findAuction)) {
-            throw new UserNotAccessibleException("경매의 판매자 또는 최종 낙찰자만 채팅이 가능합니다.");
+        if (isNotSellerAndWinner(findUser, findAuction)) {
+            throw new UserCannotAccessChatRoomException("경매의 판매자 또는 최종 낙찰자만 채팅이 가능합니다.");
         }
     }
 
-    private boolean isSellerOrWinner(final User findUser, final Auction findAuction) {
-        return findAuction.isOwner(findUser) || findAuction.isWinner(findUser, LocalDateTime.now());
+    private boolean isNotSellerAndWinner(final User findUser, final Auction findAuction) {
+        return !findAuction.isOwner(findUser) && !findAuction.isWinner(findUser, LocalDateTime.now());
```

Comment:
```suggestion
        return !(findAuction.isOwner(findUser) || findAuction.isWinner(findUser, LocalDateTime.now()));
```

하나로 묶어서 앞에 !로 부정을 해주는게 더 가독성이 좋지 않을까 생각합니다 
물론 이것도 제 개인적인 의견일 뿐입니다